scripts/export_database.pl

```diff
@@ -518,14 +521,17 @@ sub sanitize_field_content {
 
 	# only overwrite previous dump if the new one is bigger, to reduce failed runs breaking the dump.
 	my $csv_size_old = (-s $csv_filename) // 0;
-	my $csv_size_new = (-s "$csv_filename.temp") // 0;
+	# Sort lines by code, except header line
+	system("(head -1 $csv_filename.temp && tail -n +2 $csv_filename.temp | sort) > $csv_filename.temp2");
+	unlink "$csv_filename.temp";
+	my $csv_size_new = (-s "$csv_filename.temp2") // 0;
 	if ($csv_size_new >= $csv_size_old * 0.99) {
```

Comment:
```suggestion
    # guard: we replace target file only if it's big enough (to avoid replacing valid export by a broken one)
	if ($csv_size_new >= $csv_size_old * 0.99) {
```